src/audio_al.cpp

```diff
@@ -176,8 +176,10 @@ struct ALAudio::source {
 		fluid_settings_setint(settings.get(), "synth.lock-memory", 0);
 
 		synth.reset(new_fluid_synth(settings.get()), &delete_fluid_synth);
-		BOOST_VERIFY(fluid_synth_sfload(synth.get(), getenv("DEFAULT_SOUNDFONT"), 1) !=
-		             FLUID_FAILED);
+		if (fluid_synth_sfload(synth.get(), getenv("DEFAULT_SOUNDFONT"), 1) == FLUID_FAILED) {
+			Output::Error("Couldn't load soundfont\n%s.", getenv("DEFAULT_SOUNDFONT"));
+			return;
```

Comment:
@fdelapena It isn't prepared to deal with a non-working fluidsynth, so it might be better to throw an error until audio_al gets a big overhaul.